msnoise/s01scan_archive.py

```diff
@@ -157,14 +158,19 @@ def main(init=False, threads=1):
         else:
             print("No file named custom.py in the %s folder" % os.getcwd())
             return
-    for year in range(startdate.year, min(datetime.datetime.now().year, enddate.year) + 1):
+    now=startdate
+    while now <= enddate :
```

Comment:
This change will create one line per day, per station in the `folders_to_glob` list. So, at most, there will be 3 files globbed in each of these folders (3 components). The gain in time (not reading files before the start `day`) could be lost in the gigantic number of globs needed.